node/src/components/consensus/protocols/highway.rs

```diff
@@ -164,16 +167,23 @@ impl<C: Context + 'static> HighwayProtocol<C> {
             0
         };
 
+        let reduced_block_reward = match highway_config
+            .reduced_reward_multiplier
+            .checked_mul(&Ratio::from(block_reward))
+        {
+            None => u64::MAX,
+            Some(ratio) => ratio.to_integer(),
+        };
         let params = Params::new(
             seed,
             block_reward,
-            (highway_config.reduced_reward_multiplier * block_reward).to_integer(),
+            reduced_block_reward,
```

Comment:
I see those parameters weren't removed - I think they are no longer used with rewards v2, so we should be able to remove them, but that should probably be a separate ticket.

Reply:
Removed in 080a969198464c157308f2b8dffd72450c0aece4